datapurge/compat.py

```diff
@@ -2,7 +2,13 @@
 from functools import partial
 import django
 
-if django.VERSION >= (1,4):
+if django.VERSION >= (1,9):
+    from django.apps import apps
+    get_model = partial(apps.get_model)
+elif django.VERSION >= (1,7):
+    from django.db.models import get_model
+    get_model = partial(get_model)
```

Comment:
Could you explain what's the reason for having `partial` call without any additional args and kwargs, instead of just leaving plain import?

If this is only about IDE or linters style warnings (unused import) then we can simply add the `# noqa` statement that most of editors and Python linters should respect:

```python
elif django.VERSION >= (1,7):
     from django.db.models import get_model  # noqa
...
```

But it is not obligatory to change. If it is your conscious design decision (i.e. for consistency) then I'm OK with that. Just leave a comment in source code explaining the purpose of this approach.